### PR TITLE
Add support for building backend with latest OV commits

### PR DESCRIPTION
**Pull request by tanmayv25**

### CODE REVIEW

**Review by deadeyegoodwin — APPROVED** [review]

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+# Copyright 2021-2022, NVIDIA CORPORATION. All rights reserved.
```
Comment (CoderHam) [inline CMakeLists.txt:1]: & AFFILIATES

File: CMakeLists.txt [hunk 2]
```diff
@@ -45,7 +45,8 @@ option(TRITON_ENABLE_STATS "Include statistics collections in backend" ON)
 set(TRITON_BUILD_CONTAINER "" CACHE STRING "Triton container to use a base for build")
 set(TRITON_BUILD_CONTAINER_VERSION "" CACHE STRING "Triton container version to target")
 set(TRITON_BUILD_OPENVINO_VERSION "" CACHE STRING "OpenVINO version to build")
-set(TRITON_OPENVINO_BACKEND_INSTALLDIR "openvino" CACHE STRING "The directory name to install openVINO backend")
+set(TRITON_BUILD_OPENVINO_COMMIT_VERSION "" CACHE STRING "OpenVINO commit version to build")
```
Comment (CoderHam) [inline CMakeLists.txt:48]: COMMIT_SHA instead of COMMIT_VERSION?

File: src/openvino.cc [hunk 3]
```diff
@@ -729,22 +729,22 @@ ModelInstanceState::Create(
   catch (const InferenceEngine::details::InferenceEngineException& e) {
```
Comment (CoderHam) [inline src/openvino.cc:729]: Update Copyright

File: tools/gen_openvino_dockerfile.py [hunk 4]
```diff
@@ -125,6 +125,81 @@ def dockerfile_for_linux_prebuilt(output_file):
     with open(FLAGS.output, "w") as dfile:
```
Comment (CoderHam) [inline tools/gen_openvino_dockerfile.py:125]: Update Copyright

**Review by CoderHam — CHANGES_REQUESTED** [review]

**Review by GuanLuo — APPROVED** [review]
Looks good except need to address Hemant's comment

**Review by CoderHam — APPROVED** [review]

**Review by CoderHam — APPROVED** [review]

**Pull request merged**